### PR TITLE
Suggested UI text edits (SOL-1356 comment section)

### PR DESCRIPTION
**Pull request by dsego**
### Coupon Codes page

screen shot 2015-12-31 at 2 56 24 pm
### Create New Coupon page

screen shot 2015-12-31 at 2 56 48 pm
### Error messages

screen shot 2015-12-31 at 2 56 40 pm

screen shot 2015-12-31 at 2 57 08 pm

### CODE REVIEW

File: ecommerce/static/js/models/coupon_model.js [hunk 1]
```diff
@@ -18,6 +18,12 @@ define([
             ) {
         'use strict';
 
+        _.extend(Backbone.Validation.messages, {
+            required: gettext('This field is required'),
+            number: gettext('This value must be a number'),
+            digits: gettext('This value must only contain digits'),
```
Comment (clintonb) [inline ecommerce/static/js/models/coupon_model.js]: This doesn't seem necessary, given that you have the number message.

File: ecommerce/static/js/views/alert_view.js [hunk 2]
```diff
@@ -17,7 +15,11 @@ define([
             },
 
             render: function () {
-                var body = this.template({title: this.title, message: this.message});
```
Comment (clintonb) [inline ecommerce/static/js/views/alert_view.js:20]: Why has this been changed?

File: ecommerce/static/js/views/coupon_list_view.js [hunk 3]
```diff
@@ -69,12 +69,12 @@ define([
...
                             },
                             {
-                                title: gettext('Voucher Report'),
+                                title: gettext('Coupon Report'),
                                 data: 'id',
                                 fnCreatedCell: function (nTd, sData, oData) {
                                     $(nTd).html(_s.sprintf('<a href="" ' +
                                         'class="btn btn-secondary btn-small voucher-report-button" ' +
-                                        'data-coupon-id="%s"> Download Voucher Report</a>', oData.id));
+                                        'data-coupon-id="%s"> Download Coupon Report</a>', oData.id));
```
Comment (clintonb) [inline ecommerce/static/js/views/coupon_list_view.js]: Mark for translation, and use `_.template()`.

File: ecommerce/static/js/views/form_view.js [hunk 4]
```diff
@@ -49,10 +49,11 @@ define([
              * Renders alerts that will appear at the top of the page.
              *
              * @param {String} level - Severity of the alert. This should be one of success, info, warning, or danger.
-             * @param {Sring} message - Message to display to the user.
+             * @param {String} title - Title is displayed in bold.
+             * @param {String} message - Message to display to the user.
              */
-            renderAlert: function (level, message) {
-                var view = new AlertView({level: level, title: gettext('Error!'), message: message});
```
Comment (clintonb) [inline ecommerce/static/js/views/form_view.js:55]: Again, why the change for the _entire_ application, not just coupons? These are, in fact, errors on the user's part.

**Comment by clintonb** [comment]
I'm okay with changing field validation messages, but not the error alerts.

**Comment by dsego** [comment]
@clintonb  I need to change the alert message
“Error! Please complete all required fields.” to “You must complete all required fields.” 
How do you suggest I do it only for coupons?

**Comment by clintonb** [comment]
You can change the message, but why remove the "Error!" prefix? Let's just
change the message and move on. The prefix is quite minor and doesn't need
to be touched.

**Comment by dsego** [comment]
I agree. Actually all of this seems pretty minor, especially since it is an admin interface.

**Comment by mjfrey** [comment]
@dsego Leave the Error prefix and lets move on.

**Comment by mjfrey** [comment]
:+1:
